cmd/buildtools/openebs.go

```diff
@@ -57,25 +98,49 @@ var updateOpenEBSAddonCommand = &cli.Command{
 		}
 
 		// make sure we include the linux-utils image.
-		logrus.Infof("fetching the latest openebs utils image tag")
-		version, err := GetLatestGitHubRelease(c.Context, "openebs", "linux-utils")
-		if err != nil {
-			return fmt.Errorf("unable to get the latest utils image version: %v", err)
+		images = append(images, fmt.Sprintf("docker.io/openebs/linux-utils:%s", latest))
+
+		logrus.Infof("updating openebs images")
+
+		if err := ApkoLogin(); err != nil {
+			return fmt.Errorf("failed to apko login: %w", err)
 		}
-		logrus.Infof("latest github openebs utils image release: %s", version)
-		tag := strings.TrimPrefix(version, "v")
-		images = append(images, fmt.Sprintf("openebs/linux-utils:%s", tag))
 
-		logrus.Infof("fetching digest for images")
 		for _, image := range images {
-			sha, err := GetImageDigest(c.Context, image)
+			logrus.Infof("updating image %s", image)
+
+			upstreamVersion := TagFromImage(image)
+			image = RemoveTagFromImage(image)
+
+			componentName, ok := openebsImageComponents[image]
+			if !ok {
+				logrus.Warnf("no component found for image %s", image)
```

Comment:
should this be an error?

Reply:
I think this is a good idea.

Reply:
it probably should so we can know if the chart adds a new image that we aren't building with chainguard

Reply:
looks like it already fails below if the component isn't found, so...